superset-frontend/src/dashboard/util/getRevertedFilterScope.ts

```diff
@@ -18,20 +18,33 @@
  */
 import { getChartIdAndColumnFromFilterKey } from './getDashboardFilterKey';
 
+export interface FilterScopeMap {
```

Comment:
you don't need to export this (or the other interfaces, I'm going to stop notating each one)

Reply:
sure, updated